pkg/driverbuilder/builder/aliyunlinux.go

```diff
@@ -0,0 +1,107 @@
+package builder
+
+import (
+	_ "embed"
+	"fmt"
+	"github.com/falcosecurity/driverkit/pkg/kernelrelease"
+)
+
+//go:embed templates/aliyunlinux.sh
+var aliyunlinuxTemplate string
+
+// TargetTypeAlma identifies the AliyunLinux2 target.
+const TargetTypeAliyunLinux2 Type = "aliyunlinux2"
```

Comment:
Are these target names the OS ID as taken from `/etc/os-release`?
We need it to eventually integrate this with [Falco driver loader script](https://github.com/falcosecurity/falco/blob/master/scripts/falco-driver-loader)

Reply:
Thanks! So, i think we need to merge these 2 targets into a single one called `alinux`.

Reply:
> I did the same last time with AlmaLinux facepalm

:D almalinux target name is correct!
```
cat /etc/os-release | grep ID | head -n 1
ID="almalinux"
```

Reply:
it's correct because you asked me to change it last time :P https://github.com/falcosecurity/driverkit/pull/229#discussion_r1003150288

I will create a new PR for change the target to `alinux` and I will take care of updating the PR for the kernel-crawler as well

Reply:
Ahhahaha thank you for your patience! :D